### DISCUSSION TITLE
Issues with Math rendering

### DISCUSSION THREAD

**Discussion by EnochHe** [gsantner/markor#1418] [Q&A] [answered]
version 2.8.0
When writting in the markdown mode, you can't add two math formulas that both have superscripts on the same line!

for example :
$\alpha \beta ^{T}$ and $\beta\alpha ^{T}$ 

It feels annoying!!!!!!

**Comment by gsantner** [comment, marked as answer]
Nobody works actively on KaTex/Math currently. There are potential related issues at #1393 #1389.

If you know how to develop for Android you may help the project to get this improved. Thanks.

**Comment by gsantner** [comment]
KaTex was updated [here](https://github.com/gsantner/markor/actions/runs/8445971911), a test APK can be downloaded there to check if it works better. (Don't forget to check if Math is enabled/disabled accordingly in settings)
